kubernetes/base/config/incluster_config.py

```diff
@@ -109,12 +109,15 @@ def _read_token_file(self):
             ) + self._token_refresh_period
 
 
-def load_incluster_config(client_configuration=None, try_refresh_token=True):
+def load_incluster_config(client_configuration=None, try_refresh_token=True, **kwds):
     """
     Use the service account kubernetes gives to pods to connect to kubernetes
     cluster. It's intended for clients that expect to be running inside a pod
     running on kubernetes. It will raise an exception if called from a process
     not running in a kubernetes environment."""
+    for val in kds.values():
+        if val is not None:
+            raise ConfigException(f"Unimplemented named-argument {val} for incluster config.")
```

Comment:
how is ConfigException better than TypeError ?

Reply:
its only slightly better because the client code only has to catch one exception --  `ConfigException`

Secondarily, this will allow callers to: [`load_config`](https://github.com/kubernetes-client/python/blob/851dc2a0b10d1bc07826c6ade489d1bd2ef431c3/kubernetes/base/config/__init__.py#L24)

to call with a keyword arg like `context=None` and `load_incluster_config` won't bark with the `TypeError`, but if there is a `context=<val>` that's non-none, we'll get a meaningful exceptions from `load_incluster_config`